pkg/controller/controller.go

```diff
@@ -88,7 +89,7 @@ func (c *Controller) CreateVolume(in *model.VolumeSpec) (*model.VolumeSpec, erro
 	} else {
 		filterRequest = make(map[string]interface{})
 	}
-	filterRequest["size"] = in.Size
+	filterRequest["freeCapacity"] = ">= " + strconv.Itoa(int(in.Size))
```

Comment:
The filter request is too limited, you can add more filter parameter according to the definition of [PoolProperties](https://github.com/opensds/opensds/blob/development/contrib/drivers/utils/config/config.go).

Reply:
The parameters of filter can be arbitrarily filled.